system/draft/test_misc.py

```diff
@@ -1127,13 +1135,22 @@ def test_misc_hypothesis_sync(docker_setup_and_teardown, pool_handler, wallet_ha
 
 @pytest.mark.asyncio
 async def test_misc_is_1284():
```

Comment:
It's not so clear what `is_1284` means and does. I think a test name with a short description and a more detailed description in the comment would be better.

Reply:
Done.